packages/database-mongo/package.json

```diff
@@ -33,15 +33,10 @@
   "author": "Leo Pradel",
   "license": "MIT",
   "devDependencies": {
-    "@accounts/common": "^0.1.0-beta.5",
-    "@accounts/server": "^0.1.0-beta.5",
+    "@accounts/types": "^0.1.0-beta.5",
```

Comment:
should this be into dependencies?

Reply:
I guess scince it's not hosted on the DefinitelyTyped repo, We need to have it in devDeps so it cant be resolve for compilation ? What do you think?

Reply:
For me it looks like when we will install an accounts-js module sice this one is not in dependencies it will not be installed at all so typescript will fail because it will not find the type no? (I am not sure I think we will have to try this one)